lib/ruby_lsp/ruby_lsp_rails/document_symbol.rb

```diff
@@ -0,0 +1,43 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  module Rails
+    # ![Document Symbol demo](../../document_symbol.gif)
+    #
+    # The [document symbol](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_documentSymbol)
```

Comment:
Let's not include the version and always link to the latest spec.
```suggestion
    # The [document symbol](https://microsoft.github.io/language-server-protocol/specification#textDocument_documentSymbol)
```